spec/factories.rb

```diff
@@ -284,6 +299,12 @@
     staff_comments "Staff comment for factory-generated General question."
   end
 
+  factory :auto_grading_exact_option, class: Assessment::AutoGradingExactOption do
+    correct true
+    answer 'Some answer'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.